src/node/index.mjs

```diff
@@ -1,10 +1,325 @@
 /**
- * Node view for CyberChef.
- *
- * @author n1474335 [n1474335@gmail.com]
- * @copyright Crown Copyright 2017
- * @license Apache-2.0
- */
+* THIS FILE IS AUTOMATICALLY GENERATED BY src/node/config/scripts/generateNodeIndex.mjs
```

Comment:
As this is an automatically generated file, it should be in `.gitignore` otherwise it causes conflicts and merge issues.